spec/lib/lightweight_serializer/serializer_spec.rb

```diff
@@ -413,6 +413,30 @@ def translate(message)
       end
     end
 
+    context 'when called as_json with a DateTime object' do
```

Comment:
```suggestion
    context 'when called with a DateTime object' do
```
because the parent context already includes `#as_json`